src/components/QuizBuilder.ts

```diff
@@ -22,18 +22,18 @@ export class QuizBuilder {
         questions = questions.concat(this.poseQuestion(singleChoiceQuestions));
         questions = questions.concat(this.poseQuestion(textInputQuestions));
         
-        quiz.questions = this.shuffle(questions);
+        quiz.questions = this.shuffleQuestionsOrder(questions);
         return quiz;
     };
 
     /* pick questions for the quiz */
     poseQuestion(typeOfAnswers: Question[]) : Question[]{
         const picks : Question[] = [];
         for (let index = 0; index < 2; index++) {
-            let pickNumber: number = this.randomPick(typeOfAnswers);
+            let pickNumber: number = this.randomPickQuestion(typeOfAnswers);
```

Comment:
What do you think of the readability of this method name?

Reply:
oh no... the intention of my method is to describe something else